src/frameworks.js

```diff
@@ -19,18 +19,21 @@
 'use strict';
 
 var useRequireEnsure = false;
-if (typeof module !== 'undefined' && module.require) {
-  // node.js - disable worker and set require.ensure.
-  PDFJS.disableWorker = true;
-  if (typeof require.ensure === 'undefined') {
-    require.ensure = require('node-ensure');
+if (typeof module !== 'undefined' && module.exports !== 'undefined') {
+  if (module.require) {
+    // node.js - disable worker and set require.ensure.
+    PDFJS.disableWorker = true;
+    if (typeof require.ensure === 'undefined') {
+      require.ensure = require('node-ensure');
+    }
+    useRequireEnsure = true;
+  } else if (typeof __webpack_require__ !== 'undefined') {
+    // Webpack - get/bundle pdf.worker.js as additional file.
+    PDFJS.workerSrc = require('entry?name=[hash]-worker.js!./pdf.worker.js');
+    useRequireEnsure = true;
+  } else {
+    PDFJS.workerSrc = workerify './pdf.worker.js'; // jshint ignore:line
```

Comment:
Definitely. `workerify` is not a valid keyword. When browserified, it will use a transform called `workerify` that takes care of transforming this line to something like `PDFJS.workerSrc = window.URL.createObjectURL(new Blob(['worker code here']));`